input/pagecontent/StructureDefinition-au-erequesting-path-request-intro.md

```diff
@@ -0,0 +1,8 @@
+### Usage scenarios
+
+The following are supported usage scenarios for this profile:
+- Query for a request for a pathology service for a patient
+- Record or update a request for a pathology service for a patient
+
```

Comment:
With the coding introduced a STU Note must be added explaining that the curent binding to v3 is a temporary work around until v4 is available. Check out the old AU Base STU notes we used on sex and gender for ideas on wording.

Reply:
Added STU Note